components/card.css

```diff
@@ -0,0 +1,139 @@
+.card {
+  max-width: 100%;
+  background-color: var(--white-txt);
+}
+
+.card h4 {
+  margin: 0.825rem 0;
+}
+
+.card i {
+  font-size: inherit;
+}
+
+.card-title {
+  margin-bottom: 0;
+}
+
+.card .discount-percent {
+  color: #006e00;
+}
+
+/* horizontal card */
+
+.card-horizontal {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+}
+
+.card-image {
+  width: 150px;
```

Comment:
Pixels can be replaced with rems

Reply:
The card column needs to be of fixed width hence width in pixels is given.